DQMServices/Components/plugins/DQMGenericClient.cc

```diff
@@ -157,13 +157,15 @@ class FitSlicesYTool {
     TH1::AddDirectory(true);
     // ... create your hists
     TH2F* h = me->getTH2F();
-    h->FitSlicesY(nullptr, 0, -1, 0, "QNR SERIAL");
+    TF1* fitFcn = new TF1("gaus", "gaus", h->GetYaxis()->GetXmin(), h->GetYaxis()->GetXmax());
```

Comment:
Should use a different name instead of "gaus" to avoid any conflict with the global "gaus" fn, and don't add it to the global list of functions.  You also don't really need to allocate an object with new...for both, see for example

https://github.com/cms-sw/cmssw/blob/de6c103a4390562a812f8d8ea8d48d32fd9f5d2e/RecoVertex/BeamSpotProducer/src/BSFitter.cc#L340-L343